File: Assignment8/C/Program1.c
// PS - Write a program which acepts one number from user and return count of even digits

#include<stdio.h>

int CountEven(int iNo) {
    int iDigit = 0;
    int iCount = 0;

    if(iNo < 0) {
        iNo = -iNo;
    }

    while(iNo != 0) {
        iDigit = iNo % 10;
        if((iDigit % 2) == 0){
            iCount++;
        }
        iNo = iNo / 10;
    }
    return iCount;
}

int main() {
    int iValue = 0;
    int iRet = 0;

    printf("Enter number \n");
    scanf("%d", &iValue);

    iRet = CountEven(iValue);

    printf("Count of even digits are %d \n", iRet);

    return 0;
}

File: Assignment35/Java/Program1.java
// PS - Write a java program which accept number from user and return count of even digits

import java.util.*;

class DigitDemo
{
    public int CountEven(int No)
    {
        int Even = 0;
        int iDigit = 0;

        while(No != 0)
        {
            iDigit = No % 10;
            if((iDigit % 2) == 0)
            {
                Even++;
            }
            No = No / 10;
        }
        return Even;
    }
}

class Program1
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number");
        int No = sobj.nextInt();

        DigitDemo dobj = new DigitDemo();
        int iRet = dobj.CountEven(No);

        System.out.println("Count of even digits are : "+iRet);
    }
}